addon/-private/external/scheduler/policies/execution-states.js

```diff
@@ -0,0 +1,9 @@
+// TODO: change to ints?
```

Comment:
Oh I want to know this because I struggle with this all the time...

Why `int`s VS `string`s? What are the pros cons? This far I've found strings to offer better debugging for other developers and allows some consumers to avoid imports for comparing **if** they wanted to live on the edge (I know many devs who do dispite any arguments to the contrary, One of those "can't beat 'em join 'em" things. I use this pattern of constants all the time and I can never figure out if I should use ints or strings.

Reply:
Yeah I think I'd prefer to keep with strings.